content/applications/sales/sales/send_quotations/different_addresses.rst

```diff
@@ -1,75 +1,78 @@
-==========================================
-Deliver and invoice to different addresses
-==========================================
+==============================================
+Deliveries and invoices to different addresses
+==============================================
 
-With Odoo Sales, you can configure different addresses for delivery and invoicing. For some
-customers, it will be very practical to define specific billing and shipping addresses. Indeed, not
-everyone will have the same delivery location as the invoicing location.
+People and businesses often use separate addresses for billing (invoicing) and shipping (delivery)
+purposes. With the Odoo *Sales* app, contacts can have different specified addresses for delivery
+and invoicing.
 
-Activate the feature
-====================
+Settings
+========
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Customer Addresses**
-feature.
+To properly utilize multiple addresses in Odoo, go to :menuselection:`Sales app --> Configuration
+--> Settings` and scroll down to the :guilabel:`Quotations & Orders` heading. Then, check the box
+next to :guilabel:`Customer Addresses`, and click :guilabel:`Save`.
 
-.. image:: different_addresses/addresses_1.png
+.. image:: different_addresses/customer-addresses-setting.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to enable different customer addresses on Odoo Sales?
+   :alt: Activate the Customer Addresses setting.
 
-Add addresses from a quotation
-==============================
+Contact form configuration
+==========================
 
-When you create a quotation, you must add a customer. This customer can be a company or a person
-with specific billing and shipping addresses already defined and registered in the system, or it can
-be a new customer. In this case, you have to **Create and edit** the contact form for your new
-customer and link it, if necessary, to a company. In this contact form, you will be able to add,
-delete and modify invoice and delivery addresses.
+To add multiple addresses to a contact, go to :menuselection:`Sales app --> Orders --> Customers`,
+and clear any default filters from the search bar. Then, click on the desired customer to open their
+contact form.
 
-.. image:: different_addresses/addresses_2.png
+.. tip::
+   Contact forms can be accessed in the *Contacts* application, as well.
+
+From the contact form, click :guilabel:`Edit`, and then select :guilabel:`Add`, which is located
+under the :guilabel:`Contacts & Addresses` tab. Doing so reveals a :guilabel:`Create Contact` pop-up
+form, in which additional addresses can be configured.
+
+.. image:: different_addresses/contact-form-add-address.png
```

Comment:
checking: is this compressed?
it's reading as 103kb...

Reply:
Great catch! It was *not* properly compressed - but it is now. Thanks! 👍